src/radical/entk/task.py

```diff
@@ -525,12 +529,21 @@ def path(self):
     @property
     def tag(self):
         '''
-        Set the tag for the task that can be used while scheduling by the RTS
+        WARNING: It will be deprecated.
```

Comment:
The example uses `tag`, not `tags` - that should probably be changed to `tags: {colocate:...}`?

Reply:
Done